logstash-core/src/main/java/org/logstash/ObjectMappers.java

```diff
@@ -72,4 +74,22 @@ public void serialize(final RubyFloat value, final JsonGenerator generator,
             generator.writeNumber(value.getDoubleValue());
         }
     }
+
+    /**
+     * Serializer for {@link RubyBoolean} since Jackson can't handle that type natively, so we
+     * simply serialize it as if it were a {@code boolean}.
+     */
+    private static final class RubyBooleanSerializer
+        extends NonTypedScalarSerializerBase<RubyBoolean> {
+
+        RubyBooleanSerializer() {
+            super(RubyBoolean.class, true);
+        }
+
+        @Override
+        public void serialize(final RubyBoolean value, final JsonGenerator generator,
+            final SerializerProvider provider) throws IOException {
+            generator.writeBoolean(value.isTrue());
```

Comment:
value.isTrue() ?  Just double checking.. that seems like an odd way to get a boolean value.